### ISSUE TITLE
NWS geojson API returns errors

### ISSUE THREAD

**Issue by burritojustice** [tangrams/tangram#500]
**TO REPRODUCE THE ISSUE, FOLLOW THESE STEPS:**

Use https://api.weather.gov/alerts/active/region/CA as a GeoJSON source in Tangram

**RESULT:**

```Tangram v0.11.8 [error]: tile load error for active/8/40/98/8: TypeError: Cannot read property 'type' of null
    at n (https://mapzen.com/tangram/0.11/tangram.min.js:3:1299)
    at r (https://mapzen.com/tangram/0.11/tangram.min.js:3:1173)
    at new n (https://mapzen.com/tangram/0.11/tangram.min.js:3:2903)
    at Object.r [as default] (https://mapzen.com/tangram/0.11/tangram.min.js:3:2758)
    at https://mapzen.com/tangram/0.11/tangram.min.js:13:26463```

**EXPECTED RESULT:**

polygons should be displayed

**ENVIRONMENT:**

any browser

**OTHER:**

while the feed is JSON-LD, it displays as expected in geojson.io. note there are some features that do not have a geometry set.

**Comment by bcamper** [comment]
The data source fails because some features have a `null` value for the `geometry` property, which is valid GeoJSON, but is not handled by the version of geojson-vt used in the current Tangram build.

However, a newer version of geojson-vt has fixed this issue, so we will upgrade to the current 2.4.0 release in the next Tangram release.

For *tiled* data sources, we already have a check for these geometries downstream in Tangram.

**Comment by bcamper** [comment]
Upgraded in fa04ef7ad4a0c3457b279af836d7092cdb15b53d, will be in v0.12.0

**Comment by bcamper** [comment]
Also handle null geometry when `generate_label_centroids` option is on 8088f8cbdba192a65eb5fd4904b4d93e2342c970

**Comment by bcamper** [comment]
Released in [v0.12.0](https://github.com/tangrams/tangram/releases/tag/v0.12.0)
